src/components/Card/Title.tsx

```diff
@@ -3,12 +3,29 @@ import React from "react";
 
 interface TitleProps {
   paddingX?: Sprinkles["paddingX"];
+  dataTestId?: string;
+  // "title" is deprecated, use "children" instead
+  title?: React.ReactNode;
+  subtitle?: React.ReactNode;
```

Comment:
suggestion: Title should be only wrapper, imo adding this kind of prop makes it less flexible 
You can remove this props and use Title like this:
```
<Title>
    Discounts
     <Text size={4} fontWeight="light">
        3 discounts
      </Text>
</Title>
```

Reply:
I made some changes regarding this 😺